polkadot/rpc/src/lib.rs

```diff
@@ -112,7 +112,7 @@ where
 	C::Api: pallet_transaction_payment_rpc::TransactionPaymentRuntimeApi<Block, Balance>,
 	C::Api: BabeApi<Block>,
 	C::Api: BlockBuilder<Block>,
-	P: TransactionPool + Sync + Send + 'static,
+	P: TransactionPool + Sync + Send + 'static + ?Sized,
```

Comment:
```suggestion
	P: TransactionPool + 'static + ?Sized,
```
Nit. Already bound by Send+Sync. Probably there are other instance around.